### ISSUE TITLE
"Today" marker inaccurate in some views

### ISSUE THREAD

**Issue by juliaxbow** [pln-planning-tools/Starmap#189]
Date marker appears correctly in some views (see parent milestone: https://www.starmaps.app/roadmap/github.com/cryptonetlab/roadmap/issues/1) and incorrectly in others (see child milestone: https://www.starmaps.app/roadmap/github.com/cryptonetlab/roadmap/issues/21)

Screenshot:
Parent Milestone (Mostly Correct)
<img width="1523" alt="Screen Shot 2022-11-25 at 10 14 49 AM" src="https://user-images.githubusercontent.com/58958327/204038258-a257a205-2448-41ee-baa5-4026215715dd.png">

Child Milestone (Incorrect)
<img width="1543" alt="Screen Shot 2022-11-25 at 10 12 53 AM" src="https://user-images.githubusercontent.com/58958327/204038099-59013cae-b7a6-4f8b-a021-eac66f885bff.png">

**Comment by SgtPooki** [comment]
@djmcquillan this might be related to react state and updating of the today marker because the today marker positioning is based entirely on the globalTimeScaler's `getPercentage` function based on date.

**Comment by SgtPooki** [comment]
if it's related to the scale itself, it's probably not going to be a quick item.

**Comment by SgtPooki** [comment]
@djmcquillan I moved this to in progress. lmk if you're not actively working on it

**Comment by djmcquillan** [comment]
@SgtPooki as discussed we will revisit this once https://github.com/pln-planning-tools/StarMaps/pull/219 is merged

**Comment by SgtPooki** [comment]
FYI. I looked into this a bit and was unable to get it working right away but I addressed an issue with today marker rendering that was introduced in #219. 

P.S. https://www.starmaps.app/roadmap/github.com/cryptonetlab/roadmap/issues/21?mode=d3#simple shows an accurate timeline with d3, though some styling needs updating. It's so much more accurate with d3...

**Comment by SgtPooki** [comment]
This seems to be related to some special CSS-grid logic when there are stacked milestone cards. This is not reproducible when rendering a root issue that doesn't have stacked children.

**Comment by SgtPooki** [comment]
This issue is blocked by #237

**Comment by SgtPooki** [comment]
Note that this is fixed in #237 (but needs some style updates). see https://github.com/pln-planning-tools/Starmap/issues/237#issuecomment-1526912460

**Comment by SgtPooki** [comment]
Closing this in favor of https://github.com/pln-planning-tools/Starmap/issues/237
